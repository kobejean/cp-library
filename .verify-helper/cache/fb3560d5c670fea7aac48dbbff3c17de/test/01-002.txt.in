2
158968843 310416756
