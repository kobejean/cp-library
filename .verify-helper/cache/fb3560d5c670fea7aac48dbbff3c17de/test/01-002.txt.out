155208378.00000000000000000000
